app/controllers/users_controller.rb

```diff
@@ -0,0 +1,31 @@
+class UsersController < ApplicationController
+  include UsersHelper
+
+  def new
+    @user = User.new
+  end
+
+  def create
+    @user = User.new(param_filter)
+
+    if @user.save
+      redirect_to new_user_path
+    else
+      render 'new'
+    end
+  end
+
+  def edit
+    @user = User.find(params[:id])
+  end
+
+  def update
+    @user = User.find(params[:id])
+    @user.update(param_filter)
```

Comment:
Using `params_filter` is nice, however, you could consider using `user_params` in the future so your code would be more understandable especially if you have multiple controllers.